x-pack/solutions/security/plugins/security_solution/common/siem_migrations/model/api/rules/rule_migration.schema.yaml

```diff
@@ -44,52 +44,37 @@ paths:
                 additionalProperties:
                   $ref: '../../../../../common/api/detection_engine/model/rule_schema/common_attributes.schema.yaml#/components/schemas/RelatedIntegration'
 
-  ## Specific rule migration APIs
-
-  /internal/siem_migrations/rules/{migration_id}:
-    post:
+  /internal/siem_migrations/rules:
+    put:
       summary: Creates a new rule migration
-      operationId: CreateRuleMigration
+      operationId: "CreateRuleMigration"
       x-codegen-enabled: true
       x-internal: true
-      description: Creates a new SIEM rules migration using the original vendor rules provided
+      description: Creates a new rule migration and returns the corresponding migration_id
       tags:
         - SIEM Rule Migrations
-      parameters:
-        - name: migration_id
-          in: path
-          required: false
-          schema:
-            description: The migration id to create rules for
-            $ref: '../../../../../common/api/model/primitives.schema.yaml#/components/schemas/NonEmptyString'
-      requestBody:
-        required: true
-        content:
-          application/json:
-            schema:
-              type: array
-              items:
-                $ref: '../../rule_migration.schema.yaml#/components/schemas/OriginalRule'
       responses:
         200:
-          description: Indicates migration have been created correctly.
+          description: The migration was created successfully and migrationId is returned
           content:
             application/json:
               schema:
                 type: object
                 required:
                   - migration_id
                 properties:
-                  migration_id:
-                    description: The migration id created.
-                    $ref: '../../../../../common/api/model/primitives.schema.yaml#/components/schemas/NonEmptyString'
+                    migration_id:
+                      description: The migration id created.
+                      $ref: '../../../../../common/api/model/primitives.schema.yaml#/components/schemas/NonEmptyString'
 
-    put:
-      summary: Updates rules of a migrations
+  ## Specific rule migration APIs
+  /internal/siem_migrations/rules/{migration_id}:
+    patch:
+      summary: Updates rule migration data or last execution params
       operationId: UpdateRuleMigration
       x-codegen-enabled: true
       x-internal: true
-      description: Updates rules migrations attributes
+      description: Updates rules migrations data or last execution params for a single migration
```

Comment:
Can we clean all mentions of "last execution" parameters? We agreed not to allow updating them from the API.

Reply:
Done.